lib/elixir_ic/search/keyword.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ElixirIc.Search.Keyword do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "keywords" do
+    field :name, :string
+
+    field :status, Ecto.Enum,
+      default: :pending,
+      values: [pending: 0, started: 1, failed: 2, finished: 3]
+
+    field :user_id, :id
```

Comment:
Let's associate the relation instead.
```suggestion
    belongs_to :user, User
```
Also, define `has_many :keywords` for the user schema